Use `pytest.FixtureLookupError` to catch exceptions when calling `request.getfixturevalue`.

